app/views/notes/_comment.html.erb

```diff
@@ -72,11 +72,11 @@
         </a>
       <% end %>
       <% if current_user %>
-            <button style="border: 1px solid;padding: 4px 6px;" class="btn btn-outline-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
maybe the removal of '-toggle' is causing the issue with the distance between the + button and emoji menu? i can't think of anything else that would cause that issue.

Reply:
I think that distance was there before too. And, as you mentioned that those 2 enhancements are not that necessary. Totally agree on that. Let's not bother about them now and consider solving bigger bootstrap issues? Thanks!